teamsyncd/teamsync.cpp

```diff
@@ -140,14 +144,23 @@ int TeamSync::TeamPortSync::onPortChange(bool isInit)
         {
             string key = m_lagName;
             key += ":";
+
             uint32_t ifindex = team_get_port_ifindex(port);
             char ifname[MAX_IFNAME + 1] = {0};
-            key += team_ifindex2ifname(m_team, ifindex, ifname, MAX_IFNAME);
+
+            team_ifindex2ifname(m_team, ifindex, ifname, MAX_IFNAME);
+            /* netdev interface is missing */
+            if (ifname[0] == 0)
```

Comment:
When the netdev is absent. After syncd restarts, the previous netdev is no longer there. Thus it will become empty.